src/Sql/Sql/Usages/Cmdlet/GetAzureSqlInstancePoolUsage.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Usages.Cmdlet;
+using Microsoft.Azure.Commands.Sql.Usages.Models;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.Instance_Pools.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Sql.Usages
+{
+    /// <summary>
+    /// Defines the Get-AzSqlInstancePoolUsage cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstancePoolUsage",
+         DefaultParameterSetName = InstancePoolUsageDefaultParameterSet, SupportsShouldProcess = true)]
```

Comment:
@johnpaulkee we shouldn't need `SupportsShouldProcess` for a `Get` command

Reply:
Will remove.